woof-code/rootfs-petbuilds/palemoon/pinstall.sh

```diff
@@ -1,3 +1,19 @@
-echo '#!/bin/sh
-exec palemoon "$@"' > usr/local/bin/defaultbrowser
-chmod 755 usr/local/bin/defaultbrowser
+# Set defaultbrowser to run x-www-browser - set the default browser the Debian way, not the Puppy way.
+case ${DISTRO_BINARY_COMPAT} in
+    ubuntu|trisquel|debian|devuan|raspbian)
+        /usr/bin/update-alternatives --install /usr/bin/x-www-browser x-www-browser /usr/bin/palemoon 200
+        echo '#!/bin/sh
+CMD=$(command -v x-www-browser)
+[ -z "$CMD" ] && CMD="/usr/bin/palemoon"
+exec "$CMD" "$@"
+' > /usr/local/bin/defaultbrowser
+        ;;
+    *)
+        echo '#!/bin/sh
+exec /usr/bin/palemoon "$@"
+' > /usr/local/bin/defaultbrowser
+        ;;
+esac
+
+chmod 755 /usr/local/bin/defaultbrowser
```

Comment:
pinstall.sh runs without `chroot` during woof-CE, you must use relative paths - usr/local/bin/defaulbrowser, etc'

Reply:
Thanks! I'll make the changes.